gazelle/python/generate.go

```diff
@@ -88,12 +88,20 @@ func (py *Python) GenerateRules(args language.GenerateArgs) language.GenerateRes
 		}
 	}
 
+	var result language.GenerateResult
+	result.Gen = make([]*rule.Rule, 0)
+
+	pythonProjectRoot := cfg.PythonProjectRoot()
+	visibility := cfg.Visibility()
```

Comment:
Not that it really matters, but I'm curious as to why these were moved around.

Eg. why not put `if cfg.GenerateProto() {` down near L230 `collisionErrors := singlylinkedlist.New()`?

Reply:
Yeah, that's a good point, let me do that.